deb-x64/Dockerfile

```diff
@@ -68,6 +68,10 @@ RUN apt-get update \
     && apt-get install -y gcc-9 g++-9 \
     && update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-9 0
 
+# Upgrade some perl modules since some of our dependencies require some "new" features
+# that were not available in centos 7
+RUN cpanp -i List::Util
```

Comment:
Question: would it be possible to pin to a version?

Reply:
Good point. done!